### ISSUE TITLE
Add Apple's Privacy Manifest File Requirements

### ISSUE THREAD

**Issue by jmedtgit** [Flight-School/AnyCodable#82]
Hi, can you pls add the privacy manifest file to Packages.swift as required by Apple for 3rd party SDKs? https://developer.apple.com/support/third-party-SDK-requirements/

According to this requirement, all 3rd party SDKs need to have the privacy manifest file, otherwise apps that include them will be rejected for app submission starting May 1st, 2024.

Thanks.

**Comment by mithleshgomotive** [comment]
+1 to above.

**Comment by bCastro1** [comment]
Agreed, love this framework but the May 1st deadline is coming up quickly... Thanks!

**Comment by SaiChinthakuntla** [comment]
+1 to this. Would like to have new version of SDK that supports privacy manifest.
